.circleci/config.yml

```diff
@@ -371,32 +371,44 @@ jobs:
   # Publish the packages
   "publish/packages-dev":
     docker:
-      - image: docker.bintray.io/jfrog/jfrog-cli-go:latest
+      - image: amazon/aws-cli:latest
```

Comment:
Can you use the image fqdn here?

Reply:
Sure!

Good catch, I will fix it soon (also [here](https://github.com/falcosecurity/falco/pull/1577/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R396)).